mmv1/third_party/terraform/services/compute/resource_compute_backend_service_test.go.erb

```diff
@@ -1308,6 +1317,67 @@ resource "google_compute_http_health_check" "default" {
 `, serviceName, timeout, igName, itName, checkName)
 }
 
+func testAccComputeBackendService_withBackendAndIAP_enabled(
+	serviceName, igName, itName, checkName string, timeout int64) string {
+	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+  family  = "debian-11"
+  project = "debian-cloud"
+}
+
+resource "google_compute_backend_service" "lipsum" {
+  name        = "%s"
+  description = "Hello World 1234"
+  port_name   = "http"
+  protocol    = "HTTP"
+  timeout_sec = %v
+
+  backend {
+    group = google_compute_instance_group_manager.foobar.instance_group
+  }
+
+  iap {
+	enabled = true
+  }
+
+  health_checks = [google_compute_http_health_check.default.self_link]
+}
```

Comment:
@trodge any other input on this? It seems like it can't find the attribute in the test even though it's already set:

Reply:
It looks like the missing test detector is having trouble with the %v outside of quotation marks.

Reply:
I'm not sure how this would be affecting the missing test field error:
```
resource "google_compute_backend_service" "primary" {
  iap {
    enabled = # value needed
  }
}
```

since `%v` is for `timeout_sec`. I also don't think it's relevant since the use of `%v` is due to the attribute `timeout_sec` needing to be an integer and not a string